src/main/scala/org/hammerlab/guacamole/commands/VAFHistogram.scala

```diff
@@ -182,14 +181,15 @@ object VAFHistogram {
         sc.parallelize(histogramOutput).saveAsTextFile(args.output)
       } else {
         // Print histograms to standard out
-        variantAlleleHistograms.foreach(histogram =>
+        for {
+          (_, histogram) <- variantAlleleHistograms
+        } {
           histogram.toSeq.sortBy(_._1).foreach(kv => println(histogramToString(kv)))
-        )
+        }
       }
 
       if (args.cluster) {
-        val numClusters = args.numClusters
-        variantLoci.foreach(buildMixtureModel(_, numClusters))
+        buildMixtureModel(variantLoci, args.numClusters)
```

Comment:
This seems like different behavior now? Instead of building a model per sample it is just building one with all variant loci?